astropy/constants/astropyconst20.py

```diff
@@ -4,10 +4,11 @@
 for a complete listing of constants defined in Astropy.
 """
 import inspect
+from astropy.utils import find_current_module
 from . import utils as _utils
 from . import codata2014, iau2015
 
-_utils._set_c(codata2014, iau2015, inspect.getmodule(inspect.currentframe()))
+_utils._set_c(codata2014, iau2015, find_current_module())
 
 # Clean up namespace
 del inspect
```

Comment:
`inspect` and `del inspect` look useless, not sure if you need to `del find_current_module` instead ? And the same above.

Reply:
Fixed, thanks!